index.yaml

```diff
@@ -113,4 +136,4 @@ entries:
     urls:
     - https://optimizely.github.io/optimizely-agent-helm/packages/agent-1.0.2-alpha.tgz
```

Comment:
I know this is outside the changed code, but this URL works...though it's `alpha` 🤔